.github/workflows/verify-pr-labels.yml

```diff
@@ -0,0 +1,20 @@
+name: Verify Pull Request Labels
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
```

Comment:
I agree, that's too verbose in this form. 

To avoid hassling us on draft PR you can add a `if: github.event.pull_request.draft == false`  condition and add the `ready_for_review` event type to trigger it  

Removing `synchronize` would help too, the need for a label does not change when you push to the branch.